src/Services/QueuedJobService.php

```diff
@@ -1074,7 +1088,9 @@ public function getJobListFilter($type = null, $includeUpUntil = 0)
      */
     public function runQueue($queue)
     {
-        $this->checkJobHealth($queue);
+        if (!self::config()->disable_health_check) {
```

Comment:
```diff
-         if (!self::config()->disable_health_check) {
+         if (!$this->config()->get('disable_health_check')) {
```

Also note my suggestion to flip this argument around so it'd be `if ($this->config()->get('health_check')) { ...`